### PR TITLE
Make logging more verbose in the label syncer

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

Fixes https://github.com/kubernetes/test-infra/issues/5818
/assign @fejta

### CODE REVIEW

File: label_sync/main.go [hunk 1]
```diff
@@ -151,10 +151,10 @@ func LoadRepos(org string, gc *github.Client, filt filter) (RepoList, error) {
 func LoadLabels(gc *github.Client, org string, repos RepoList) (*RepoLabels, error) {
 	rl := RepoLabels{}
 	for _, repo := range repos {
-		logrus.Infof("Get labels in %s/%s", org, repo.Name)
+		logrus.WithField("org", org).WithField("repo", repo.Name).Info("Listing labels for repo")
```
Comment (fejta) [inline label_sync/main.go]: for repo seems redundant here?

File: label_sync/main.go [hunk 2]
```diff
@@ -423,30 +425,29 @@ func main() {
...
-		y, _ := yaml.Marshal(updates)
-		fmt.Println(string(y))
-	}
+
+	y, _ := yaml.Marshal(updates)
+	logrus.Debug(string(y))
 
 	if *dry {
-		logrus.Infof("No real update labels in --dry-run")
+		logrus.Infof("Running in --dry-run mode, no mutations made")
```
Comment (fejta) [inline label_sync/main.go]: can we switch this to `--wet`? 😈

Comment (stevekuznetsov) [reply label_sync/main.go]: Not sure I understand -- would you like me to change the flag name>

File: label_sync/main.go (hunk 1 above)
Comment (stevekuznetsov) [reply label_sync/main.go]: ACK

File: label_sync/main.go (hunk 2 above)
Comment (fejta) [reply label_sync/main.go]: I'm mostly joking, although I'd rather have `--foo` flags than `--unfoo=false` (thus `--wet` rather than `--dry-run=false`)

Comment (stevekuznetsov) [reply label_sync/main.go]: Yeah, I did get tripped up by `--dry-run=false`. I'll tackle that in a separate PR.

**Pull request merged**